compose/ui/ui-test/build.gradle

```diff
@@ -147,10 +147,17 @@ if (AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 implementation(libs.kotlinTest)
             }
 
-            skikoMain {
-                dependsOn(commonMain)
+            skikoMain.dependencies {
+                implementation(project(":lifecycle:lifecycle-common"))
+                implementation(project(":lifecycle:lifecycle-runtime"))
```

Comment:
Shouldn't it be in common?

Reply:
It isn't used in Android or commonMain, so we should keep it here

Reply:
They are not required in common, as I introduced changes in `skikoMain`/`skikoTest` only, so I decided to include dependencies only for these source roots.

Reply:
I'd like to see some common API at some point, but it requires changes in AOSP first